genesyscloud/resource_genesyscloud_recording_media_retention_policy.go

```diff
@@ -972,7 +972,7 @@ func getAllMediaRetentionPolicies(_ context.Context, clientConfig *platformclien
 
 	for pageNum := 1; ; pageNum++ {
 		const pageSize = 100
-		retentionPolicies, _, getErr := recordingAPI.GetRecordingMediaretentionpolicies(pageSize, pageNum, "", []string{}, "", "", "", true, false, false, 365)
+		retentionPolicies, _, getErr := recordingAPI.GetRecordingMediaretentionpolicies(pageSize, pageNum, "", []string{}, "", "", "", true, false, false, 0)
```

Comment:
Hey @PrinceMerluza ,

Why did you chose a value of 0 here.  I could not find any documentation for this API about setting this value to zero so I just wanted to see you reasoning behind setting this to zero.

Reply:
Hi John, so technically the correct API call really would be setting this to 'null/nil' and not including `deleteDaysThreshold` as a query parameter in the actual request. However this API method only allows `int` and so I tested if the behavior would be the same.

And upon testing setting it to `0` is the same as ignoring it since it will include all policies that have `deleteDaysThreshold  > 0` ie. include all policies since `deleteDaysThreshold` is required to be a positive integer.